python/test/test_algo_bench.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import contextlib
+import json
+import os
+import sys
+import time
+from collections import namedtuple
+from datetime import datetime
+
+import numpy as np
+import pytest
+import pytz
+from pyarrow import Schema
+
+import katana.benchmarking.bench_python_cpp_algos as katbench
```

Comment:
Don't use `as`. It hides what you are referring to. You can either say `bench_python_cpp_algos` below or import the specific values you need from that module. (`np` is an exception since it's so well known, though honestly I personally don't like it either.)